packages/empty/README.md

```diff
@@ -0,0 +1,10 @@
+# @joplin/empty
+
+An empty package. This package can be used to exclude certain dependencies from build.
+
+For example, the `canvas` dependency is an optional dependency of `pdfjs-dist`. However, it isn't used by Joplin and can cause build to fail in certain environments. The `@joplin/empty` package can exclude `canvas` from the build by adding a resolution to `resolutions` in the top-level `package.json`. For example, resolving `canvas@npm:^2.11` to `file:./packages/empty/`.
```

Comment:
Previously, `canvas` was excluded from build by manually modifying `yarn.lock`. This no longer works with Yarn v4. As a result, this pull request takes the [approach suggested here](https://github.com/yarnpkg/yarn/issues/4611#issuecomment-1370284462) and maps `canvas@npm:^2.11.2` to a new `@joplin/empty` package.